.spelling

```diff
@@ -1,6 +1,16 @@
+Bodgenhurst
+Brokenhurst
+Collyer's
 element14
 Farnell
 kickstart
+knockouts
+Livestream
```

Comment:
Is the word "livestream" required to always be presented with a capital letter? If not, it doesn't need to be capitalised here (which allows the spellchecker to accept the word regardless of whether it is capitalised or not).

Reply:
aha, neat: be50b9e